src/alcaramba/Drawable/Town/Town.gd

```diff
@@ -165,9 +235,49 @@ func walls_between(card1: TileCard, card2: TileCard, direction: String)-> int:
 
 	return 0
 
+func walls_between1(pos1, pos2)-> int:
```

Comment:
Please find a better, more explicit name for `walls_between1()`. It is also easy to overlook this on callees